e2e/testsuite/testsuite.go

```diff
@@ -31,6 +31,8 @@ const (
 	ChainARelayerName = "rlyA"
 	// ChainBRelayerName is the name given to the relayer wallet on ChainB
 	ChainBRelayerName = "rlyB"
+
+	emptyLogs = "[]"
```

Comment:
should we add a super basic godoc for this?

Reply:
yep good idea I'll add one